docs/traffic-distribution.md

```
@@ -0,0 +1,209 @@
+```bash
+LLM_ROUTING_STRATEGY=weighted-shuffle
+CEREBRAS_WEIGHT=5
+GROQ_WEIGHT=1
+NVIDIA_NIM_WEIGHT=1
+```
+
+---
+
+## Observability
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)